riscv-asm.md

```diff
@@ -540,7 +568,10 @@ fail_msg:
 
 Pseudoinstruction            | Base Instruction(s)                                           | Meaning   | Comment
 :----------------------------|:--------------------------------------------------------------|:----------|:--------|
-la rd, symbol                | auipc rd, symbol[31:12]; addi rd, rd, symbol[11:0]            | Load address
+la rd, symbol                | auipc rd, symbol[31:12]; addi rd, rd, symbol[11:0]            | Load address | Without -fPIC
+la rd, symbol                | auipc rd, symbol@GOT[31:12]; l{w\|d} rd, symbol@GOT[11:0]\(rd\) | Load address | With -fPIC
```

Comment:
Wasn't sure what syntax to use here; we're already sloppy about the PC-relative nature of symbol above, and not using proper assembly syntax, so I figured something like this was most in keeping for giving the high-level idea without the specifics that would make the table even more unwieldy.

Reply:
Methinks that the canonical pseudo functions should be used instead in order to avoid confusion to users.